### PR TITLE
day 4 homework

### PR DESCRIPTION
**Pull request by RobertStoelhorst**
_If you had no problems you can just leave it empty_

> How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
A: 9

> Was there anything that you struggled with?
A: was very hard for me so many different directions my brain was going and struggled to focus. I'm a lot more excited to learn about peoples methods his morning. It's a love hate relationship with mathmatics.

> Is there any particular code you want me to look into?
nothing that stands out it all needs work

> Is there anything that you'd like some further information on?
functions are my biggest weekness at the moment, going to spend a lot more time on the weekend going over the basics.

> Roughly how long did it take?
I spent about 4-5 hours on this and that burned me out.

### CODE REVIEW

File: Robert Stoelhorst/week01/day_04/functions/homework.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const areaOfRectangle = function (x, y) {
+  const area = x * y;
+  const perimeter = x * 2 + y * 2;
+  if (x === y)
+return (` This is a square and the area of this square is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
+  if (x !== y)
+return (` The area of this rectangle is ${ area }cm2 and it's perimeter is ${ perimeter }cm`)
+}
+  areaOfRectangle(10, 10)
```
Comment (CamillaCdC) [inline Robert Stoelhorst/week01/day_04/functions/homework.js:17]: I know this function works, but if I'm being particular about your indentation and syntax it should look like: 

const areaOfRectangle = function (x, y) {
  const area = x * y;
  const perimeter = x * 2 + y * 2;
  if (x === y) {
      return (` This is a square and the area of this square is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
  }
  if (x !== y) {
      return (` The area of this rectangle is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
  }
}

File: Robert Stoelhorst/week01/day_04/functions/homework.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    // // const areaObtuse = ((y * y))
+    if (x === y && y === z)
+    return (`This is an Equilateral triangle and its area is `);
+    else if ((x * x) > ((y * y) + (z * z)))
+    return (`This is an Obtuse triangle and its area is `);
+    else if (x < y && y === z)
+    return (`This is an Isoscelese triangle and its area is `);
+    else
+    return (`This has gone beyond 180 degrees its definatly not a triangle`)
+  }
```
Comment (CamillaCdC) [inline Robert Stoelhorst/week01/day_04/functions/homework.js:40]: same as comment above about indentation. Also please make sure you include the { } in your if/elseif/else statements. You will run into issues down the track if you don't get into the habit of including these.

**Comment by CamillaCdC** [comment]
Don't stress too much about the math stuff. It is important because there is always a bit of math behind programming, but depending on where you end up working there will be less geometry!

**Comment by CamillaCdC** [comment]
Waiting for submission answers for MTA Lab (check Slack message)

**Comment by RobertStoelhorst** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
A: 7

Was there anything that you struggled with?
A: I wouldn't say I struggled, just took my time to try and get things working the way I wanted them to, I get caught up in my perfectionism. still needs work 
 
Is there any particular code you want me to look into?
nothing that stands out it, just trying to learn more about responsive web design

Is there anything that you'd like some further information on?
at the moment it seems easy enough to find answers online to the things I need

Roughly how long did it take?
I spent about 2.5 hours on this to get it working as is and played with the code for another hour to see how things respond.

**Comment by CamillaCdC** [comment]
I understand the getting caught up in perfectionism!! I fall into the same trap sometimes. Its not necessarily a bad thing, but work at it piece by piece and get the overall thing working before you worry to much about it being "perfect". As I've said before - there's often so many ways to go about a program so there isn't one "perfect solution". 

I'm glad you're able to find resources online - that is definitely a skill! 

We'll be going more into responsive design later on. 

Please keep indentation in mind when writing your code. Its a little inconsistent. I know it seems trivial now, but its becomes very important when you write bigger programs and start working in teams.

**Pull request merged**